src/coreComponents/dataRepository/Group.cpp

```diff
@@ -268,6 +268,18 @@ void Group::initializationOrder( string_array & order )
   }
 }
 
+void Group::initialize_postMeshGeneration()
+{
+  string_array initOrder;
```

Comment:
So what's our policy about `string_array`? Should we use it or `array1d< string >`?

Reply:
we have no policy. I typically just use `array1d<string>` nowadays. I think it makes more sense due to the use of `arrayView`.